api/backend/blueflood/blueflood.go

```diff
@@ -140,9 +140,9 @@ func (b *Blueflood) fetchSingleSeries(metric api.GraphiteMetric, sampleMethod ap
 	}
 
 	params := url.Values{}
-	params.Set("from", strconv.FormatInt(timerange.Start*1000, 10))
-	params.Set("to", strconv.FormatInt(timerange.End*1000, 10))
-	params.Set("resolution", bluefloodResolution(timerange.Resolution))
+	params.Set("from", strconv.FormatInt(timerange.Start(), 10))
+	params.Set("to", strconv.FormatInt(timerange.End(), 10))
+	params.Set("resolution", bluefloodResolution(timerange.Resolution()))
```

Comment:
I'm not sure about these changes, exactly, since I'm not sure what units are involved in the blueflood backend portion. `.Start()` and `.End()` should be interpreted as milliseconds.